apps/pie-docs/src/_includes/content-page.njk

```diff
@@ -1,25 +1,34 @@
 {% set heroSrc = ['/assets/img/heroes/', heroImageSrc] | join | url %}
 
 <div class="c-contentPage">
-    <div
-        class="c-contentPage-header {% if heroImageSrc %}c-contentPage-header--hero{% endif %}"
-        {% if heroImageSrc %} style="--hero-img-src: url({{heroSrc}})" {% endif %}>
+    <div class="c-contentPage-header {% if heroImageSrc %}c-contentPage-header--hero{% endif %}">
         <div class="c-content-container">
-            <h1 class="c-contentPage-header-heading alpha--standout">
-                {{ title }}
-            </h1>
-            <p class="c-contentPage-header-description">
-                {{ description }}
-            </p>
+            <div>
+                <h1 class="c-contentPage-header-heading alpha--standout">
+                    {{ title }}
+                </h1>
+                <p class="c-contentPage-header-description">
+                    {{ description }}
+                </p>
+                {% if heroCtaTitle and heroCtaHref %}
+                    <a class='o-btn' href={{ heroCtaHref | url }}>{{ heroCtaTitle }}
+                    </a>
+                {% endif %}
+            </div>
+            {% if heroImageSrc %}
+                <div>
+                    <img class="u-showAboveMid" src={{ heroSrc }} alt="{{ heroImageAlt }}"/>
```

Comment:
Is an alt necessary here? It's a decorative image being displayed next to a description of the website, so I wonder what an alt is really adding to the screen reader UX here

Reply:
wondering if it should be an alt vs defining the role is decoration etc

Reply:
awesome! cheers